src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -348,6 +349,14 @@ delegate_declaration
   : attribute_list* modifier* 'delegate' type identifier_token type_parameter_list? parameter_list type_parameter_constraint_clause* ';'
   ;
 
+extension_container
+  : attribute_list* modifier* syntax_token type_parameter_list? '(' receiver_parameter ')' type_parameter_constraint_clause* '{' member_declaration* '}'
+  ;
+
+receiver_parameter
+  : attribute_list* modifier* type identifier_token?
```

Comment:
we should allow a parameter_syntax.  it will make it much easier ot recover and represent throughout the system.  (similar to how even `a => a` uses a parameter_syntax).  

overfitting syntax is often problematic.  both for normal typing cases, and for expanding on this in the future.